src/docfx/build/xref/XrefResolver.cs

```diff
@@ -188,6 +200,28 @@ public XrefMapModel ToXrefMapModel(bool isLocalizedBuild)
             return model;
         }
 
+        // xrefs marked ValidateExternalXrefs, including uids defined internal or external repo
+        // this function return net xrefs, which defined external repo
+        private ExternalXref[] GetNetExternalXrefs()
+        {
+            if (!_valdiateExternalXrefList.IsValueCreated)
+            {
+                _ = _valdiateExternalXrefList.Value;
+            }
+
+            return _valdiateExternalXrefList.Value.Where(uid => !_internalXrefMap.Value.ContainsKey(uid.Value)).GroupBy(uid => uid.Value).Select(uidGroup =>
```

Comment:
To see if an xref is external, check if the resolved xref spec is `ExternalXrefSpec` in `JsonSchemaTransformer.cs` line 421

Reply:
OK